Adyen/UI/Form/Items/Value/Validatable/FormValidatableValueItemView.swift

```diff
@@ -103,9 +103,20 @@ open class FormValidatableValueItemView<ValueType, ItemType: FormValidatableValu
     
     private func hideAlertLabel(_ hidden: Bool, animated: Bool = true) {
         guard hidden || alertLabel.text != nil else { return }
+        if !hidden {
+            triggerValidationErrorIfNeeded()
+        }
         alertLabel.adyen.hide(animationKey: "hide_alertLabel", hidden: hidden, animated: animated)
     }
     
+    // this function can be used to populate the alert label with the correct invalid message
```

Comment:
The message is not super clear to me without any context rn - maybe it's not even needed as the function name is very self explainatory